bot/exts/moderation/infraction/superstarify.py

```diff
@@ -208,18 +208,19 @@ async def _pardon_action(self, infraction: _utils.Infraction) -> t.Optional[t.Di
             )
             return {}
 
+        log_text = {"Member": format_user(user)}
+
         # DM the user about the expiration.
```

Comment:
You should move this inside the if statement like in the other file.

Reply:
Unlike the other function, this one has an early exit with `if not user: ... return`. Your suggestion would call for a refactor of the function, resulting in a more significant diff. I didn't want to change more than necessary.

Reply:
To clarify, I mean the comment. You should move the comment down a line to be consistent.

Reply:
Oh. That's such a minor thing, I don't think it's worthwhile. Maybe if I had another reason to be modifying that area of code now, I'd throw in this change.

Reply:
Refering to our other conversation, this could be the same reasoning as to why capitalization is so inconsistent.

Reply:
That's just because no one noticed when reviewing the PRs which introduced those messages. Furthermore, there's an important difference: those messages are user-facing, but comments such as this one are not. Outside of readability/ambiguity reasons, I think worrying about the placement of comments is being too pedantic; someone doesn't need to go out of their way to address such issue.

Reply:
I won't continue arguing about this, but code is read more times than it is written. It most likely will never be changed when this is merged, so now is the time to change it.

Reply:
You're right, but in this case, I think that's okay; moving this comment would be so inconsequential. If we're concerned with consistency, the function should undergo a more significant refactor as I originally mentioned. In fact, I will later make a separate PR to make the message capitalisation consistent, and I can also refactor this function in that PR.